src/main/resources/application-local.yml

```diff
@@ -27,4 +27,9 @@ gridsuite:
     -
       name: voltage-init-server
       base-uri: http://localhost:5038
-  
+    -
```

Comment:
Question: Do we have documentation somewhere where we document which micro-service talks to which? If yes, we need to update it as well.

Reply:
Maybe this shema  : https://github.com/gridsuite/documentation. But it's outdated a long time ago . 
This file is only for local devs purposes.